src/bitset.hpp

```diff
@@ -103,24 +106,28 @@ class Bitset {
   std::optional<uint32_t> SingletonOption() const;
   // Output as string of comma-separated indices.
   std::string ToVectorOfSetBitsAsString() const;
+  // Returns a new Bitset with size equal to `idx_table`'s size. Each entry
+  // of the new bitset is determined as follows:
+  // - If the `idx_table` entry is an integer i, then the value is the ith
+  //   entry of `bitset`.
+  // - If the entry is nullopt, then the value is False (i.e. zero).
+  static Bitset Remap(const Bitset &bitset, const SizeOptionVector &idx_table);
 
   // ** Clade / MultiClade Methods
   // These methods require bitsets to represent "clades". A clade is an expression of a
-  // subset of a taxon set.  The size of the taxon set is equal to the size of the
+  // subset of a taxon set.  The size of the total taxon set is equal to the size of the
   // clade's bitset, with each bit index representing a inclusion/exclusion of a
   // specific member of that taxon set.
   //
   // There are bitset "types" composed of multiple clades (here, called a
-  // MultiClade). Subsplits are composed of two clades and PCSPs are composed of three
+  // MultiClade). Subsplits are composed of two clades and Edges are composed of three
```

Comment:
```suggestion
  // MultiClade). Subsplits are composed of two clades and PCSPs are composed of three
```

Maybe have a little search?